src/main/java/util/DatabaseUtil.java

```diff
@@ -0,0 +1,205 @@
+package util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.log4j.ConsoleAppender;
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.apache.log4j.PatternLayout;
+import configuration.DatabaseConfiguration;
+
+public class DatabaseUtil {
```

Comment:
It should be named `DatabaseExecutor`
And please make it can be used to `Dependency Injection`